users/src/main/java/io/crate/operation/auth/HostBasedAuthentication.java

```diff
@@ -36,11 +36,11 @@
 
 public class HostBasedAuthentication implements Authentication {
 
-    private static final String DEFAULT_AUTH_METHOD = "trust";
-    private static final String KEY_USER = "user";
-    private static final String KEY_ADDRESS = "address";
-    private static final String KEY_METHOD = "method";
-    private static final String KEY_PROTOCOL = "protocol";
+    public static final String DEFAULT_AUTH_METHOD = "trust";
```

Comment:
I think they should be public because these are hardcoded in a lot of places which makes it difficult to change them. I avoided to hardcode in my tests. Generally, `static final` should be safe to be made public.